lib/shenzhen/commands/build.rb

```diff
@@ -87,6 +87,36 @@
     log "xcrun", "PackageApplication"
     abort unless system %{xcrun -sdk #{@sdk} PackageApplication -v "#{@app_path}" -o "#{@ipa_path}" --embed "#{options.embed || @dsym_path}" #{"-s \"#{options.identity}\"" if options.identity} #{'1> /dev/null' unless $verbose}}
 
+    # Determine whether this is a Swift project and, eventually, the list of libraries to copy from
+    # Xcode's toolchain directory since there's no "xcodebuild" target to do just that (it is done
+    # post-build when exporting an archived build from the "Organizer").
+    @ipa_swift_frameworks = Dir["#{@app_path}/Frameworks/libswift*"]
+
+    if not @ipa_swift_frameworks.empty? then
+      Dir.mktmpdir do |tmpdir|
+        # Copy all necessary Swift libraries to a temporary "SwiftSupport" directory so that we can
+        # easily add it to the .ipa later.
+        swift_support = File.join(tmpdir, "SwiftSupport")
+
+        Dir.mkdir(swift_support)
+
+        xcode = `xcode-select --print-path`.strip
+
+        @ipa_swift_frameworks.each do |path|
+          framework = File.basename(path)
+
+          FileUtils.copy_file("#{xcode}/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/#{@sdk}/#{framework}", File.join(swift_support, framework))
+        end
+
+        # Add "SwiftSupport" to the .ipa archive
+        pwd = Dir.pwd
```

Comment:
Right! I didn't see that while looking up the docs. I know enough Ruby to be dangerous but it's not really my cup of tea :)